docs/templates/plugins.rst.j2

```diff
@@ -1,4 +1,7 @@
 :tocdepth: 0
+{% if STEP == 'prepare-feature' %}
+:orphan:
+{% endif %}
```

Comment:
Why is the `:orphan:` used here? When opening the page, left menu is closed (no menu item highlighted) so it's not clear under which section user currently is.

Reply:
It's gone now.